client/requestUtil.js

```diff
@@ -1,27 +1,103 @@
 'use strict'
 
+const MAX_RETRIES = 1
+const NONCE_COUNTER = 0
```

Comment:
this should be imported from client/config.js so it doesn't need to be edited in more than one place when it changes